### PR TITLE
Comment on wiki activity stream

### PR DESCRIPTION
**Pull request by esmaoui**

### CODE REVIEW

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiCommentActivityStreamTestIT.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+/**
+ * Created by esmaoui on 23/03/2018.
+ */
+
+
+
+@Tag("sniff")
+@Tag("wiki")
+public class WikiCommentActivityStreamTestIT extends Base{
```
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiCommentActivityStreamTestIT.java]: you can add this TC in WikiPublishActivityTestIT.java .

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiCommentActivityStreamTestIT.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    HomePagePlatform homePagePlatform;
+    WikiManagement wikiManagement;
+    RichTextEditor richTextEditor;
+    SourceTextEditor sourceTextEditor;
```
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiCommentActivityStreamTestIT.java]: useless variable

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiCommentActivityStreamTestIT.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        manageLogInOut = new ManageLogInOut(this);
+        if ($(ELEMENT_SKIP_BUTTON).is(exist)) {
+            $(ELEMENT_SKIP_BUTTON).click();
+        }
+
+        manageLogInOut.signInCas( DATA_USER1 , DATA_PASS2 );
+
+}
+    /*
+     bug WIKI-1392
```
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiCommentActivityStreamTestIT.java]: use @Tag("WIKI-1392")

File: commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java [hunk 4]
```diff
@@ -82,7 +82,7 @@ public void afterEach() {
     Screenshots.finishContext();
   }
 
-  @AfterEach
+ @AfterEach
```
Comment (thomasdelhomenie) [inline commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java]: Indentation not good.

Comment (aboughdiri) [reply commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java]: done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiCommentActivityStreamTestIT.java (hunk 3 above)
Comment (aboughdiri) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiCommentActivityStreamTestIT.java]: done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiCommentActivityStreamTestIT.java (hunk 1 above)
Comment (aboughdiri) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiCommentActivityStreamTestIT.java]: Done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiCommentActivityStreamTestIT.java (hunk 2 above)
Comment (aboughdiri) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiCommentActivityStreamTestIT.java]: done

**Pull request merged**